Training Modules/04 Simple Implentations/4.01 QR Code Generator/README.md

```diff
@@ -0,0 +1,8 @@
+# QR Code Generator
```

Comment:
Could you add this just under the heading?

```
_Originally contributed by [@adityaSINGHrox](https://github.com/adityaSINGHrox)_
```

Reply:
i'll do that